kubernetes/cache/cache.go

```diff
@@ -204,6 +207,48 @@ func (in *kialiCacheImpl) IsAmbientEnabled(cluster string) bool {
 	return check
 }
 
+// IsAmbientEnabled checks if the istio Ambient profile was enabled
+// by checking if the ztunnel daemonset exists on the cluster.
+func (in *kialiCacheImpl) GetZtunnel(cluster string) models.PodsList {
```

Comment:
nit: the comment is for a different function here.

Why not just return the `ztunnel` pods themselves?